examples/coap/src/bin/coapserver-coaphandler.rs

```diff
@@ -2,6 +2,8 @@ use hexlit::hex;
 use lakers::*;
 use lakers_crypto::Crypto;
 
+use coap_message::{Code, MinimalWritableMessage, MutableWritableMessage, ReadableMessage};
+use coap_message_utils::{Error, OptionsExt as _};
```

Comment:
`Error` is very generic, maybe alias to something like `CoapError`?

Reply:
It's the only Error type this code touches; I'd prefer not to name types around.

On the long run, I expect that this module grows its own error type (that is a union of coap_message_utils::Error and the two wrapped errors), and implements the CoAP error rendering. Then, that will be called Error, and there will be a From<coap_message_utils::Error> to it (with coap_message_utils::Error explicitly named rather than imported because it will be rare by then). But so far, every kind of error encountered in this program is eventually converted into this type.

Reply:
Sounds good.